src/routes/RequestsRoute.js

```diff
@@ -728,34 +738,42 @@ class RequestsRoute extends React.Component {
         >
           <FormattedMessage id="ui-requests.exportSearchResultsToCsv" />
         </Button>
-        <Button
-          buttonStyle="dropdownItem"
-          id="exportExpiredHoldsToCsvPaneHeaderBtn"
-          disabled={servicePointId && isEmpty(requests)}
-          onClick={() => {
-            onToggle();
-            this.exportExpiredHoldsToCSV();
-          }}
-        >
-          <FormattedMessage
-            id="ui-requests.exportExpiredHoldShelfToCsv"
-            values={{ currentServicePoint: servicePointName }}
-          />
-        </Button>
-        <PrintButton
-          buttonStyle="dropdownItem"
-          id="printPickSlipsBtn"
-          disabled={isEmpty(pickSlips)}
-          template={this.getPrintTemplate()}
-          dataSource={convertToSlipData(pickSlips, intl, timezone, locale)}
-          onBeforePrint={onToggle}
-        >
-          <FormattedMessage
-            id="ui-requests.printPickSlips"
-            values={{ sp: servicePointName }}
-          />
-        </PrintButton>
-      </>
+        {
+          !pickSlipsArePending ?
+            <LoadingButton>
+              <FormattedMessage id="ui-requests.pickSlipsLoading" />
+            </LoadingButton> :
+            <>
+              <Button
+                buttonStyle="dropdownItem"
+                id="exportExpiredHoldsToCsvPaneHeaderBtn"
+                disabled={servicePointId && isEmpty(requests)}
+                onClick={() => {
+                  onToggle();
+                  this.exportExpiredHoldsToCSV();
+                }}
+              >
+                <FormattedMessage
+                  id="ui-requests.exportExpiredHoldShelfToCsv"
+                  values={{ currentServicePoint: servicePointName }}
+                />
+              </Button>
+              <PrintButton
+                buttonStyle="dropdownItem"
+                id="printPickSlipsBtn"
+                disabled={isEmpty(pickSlips)}
+                template={this.getPrintTemplate()}
+                dataSource={convertToSlipData(pickSlips, intl, timezone, locale)}
+                onBeforePrint={onToggle}
+              >
+                <FormattedMessage
+                  id="ui-requests.printPickSlips"
+                  values={{ sp: servicePointName }}
+                />
+              </PrintButton>
+            </> //
```

Comment:
Are there comments on these lines? I'm not sure what the // are there for.

Reply:
could catch @Baroquem. I added them there because my editor gets confused with `</>` and screws up the color theme. I just cleaned that up.